Fatras/include/ActsFatras/Kernel/Simulation.hpp

```diff
@@ -49,13 +49,22 @@ struct SingleParticleSimulation {
   interactions_t interactions;
   /// Selector for surfaces that should generate hits.
   hit_surface_selector_t selectHitSurface;
+  /// Wrapped logger for debug output.
+  Acts::LoggerWrapper Logger = Acts::getDummyLogger();
```

Comment:
this does not comply with our naming conventions, it must not be capitalised and should be called probably `loggerWrapper`.